diff --git a/cmake/Deps.cmake b/cmake/Deps.cmake
--- a/cmake/Deps.cmake
+++ b/cmake/Deps.cmake
@@ -85 +85 @@
-    URL https://github.com/google/boringssl/archive/refs/tags/0.20251124.0.tar.gz
+    URL https://codeload.github.com/google/boringssl/tar.gz/refs/tags/0.20251124.0
@@ -105 +105 @@
-    URL https://github.com/madler/zlib/archive/refs/tags/v1.3.1.tar.gz
+    URL https://codeload.github.com/madler/zlib/tar.gz/refs/tags/v1.3.1
@@ -138 +138 @@
-        URL https://github.com/protocolbuffers/protobuf/archive/refs/tags/v21.12.tar.gz
+        URL https://codeload.github.com/protocolbuffers/protobuf/tar.gz/refs/tags/v21.12
